datafusion/core/tests/dataframe/describe.rs

```diff
@@ -40,12 +40,12 @@ async fn describe() -> Result<()> {
         "+------------+-------------------+----------+--------------------+--------------------+--------------------+--------------------+--------------------+--------------------+-----------------+------------+-------------------------+--------------------+-------------------+",
```

Comment:
I believe these precision changes relate to https://github.com/apache/arrow-rs/pull/5100

Reply:
This seems like a better result to me (less numeric stability)